web_src/js/components/DashboardRepoList.vue

```diff
@@ -445,3 +447,33 @@ export function initDashboardRepoList() {
 export default sfc; // activate the IDE's Vue plugin
 
 </script>
+<style scoped>
+ul {
+  list-style: none;
+  margin: 0;
+  padding-left: 0;
+}
+
+ul li {
+  padding: 0 10px;
+}
+
+ul li:not(:last-child) {
+  border-bottom: 1px solid var(--color-secondary);
+}
+
+.repo-list-link {
+  padding: 6px 0;
+  gap: 6px;
+  min-width: 0; /* for text truncation */
+}
+
+.repo-list-link .svg {
+  color: var(--color-text-light-2);
+}
+
+.repo-list-icon {
+  min-width: 16px;
```

Comment:
Does this `min-width` conflicts with the svg size when `repoIconSize=14`?

Reply:
The one 14px SVG will be 16x14, but it still looks fine and is actually preferred so the content does not shift.